internal/config/seccomp/oci_artifact.go

```diff
@@ -0,0 +1,95 @@
+package seccomp
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/containers/image/v5/docker/reference"
+	"oras.land/oras-go/v2"
+	"oras.land/oras-go/v2/content/file"
+	"oras.land/oras-go/v2/registry/remote"
+
+	"github.com/cri-o/cri-o/internal/log"
+	"github.com/cri-o/cri-o/pkg/annotations"
+)
+
+const (
+	defaultTag                = "latest"
+	defaultSeccompProfilePath = "seccomp.json"
+)
+
+func tryPullSeccompOCIArtifact(
```

Comment:
Yes, but we can probably replace ORAS with the content of https://github.com/containers/common/pull/1833. I'm going to double check how it can be done.

Reply:
Done